frontend/src/i18n/en-US.json

```diff
@@ -155,6 +155,11 @@
   "initialSetup.accountKey": "Account Key",
   "initialSetup.submit": "Finish Setup",
 
+  "licenseAlert.title": "Attention needed",
+  "licenseAlert.noRemainingSeats": "Your Cryptomator Hub license has exceeded the number of available seats. {0} to renew it to manage and unlock your vaults again.",
```

Comment:
I would reorder the second sentence a bit:
"Your Cryptomator Hub... . Renew it in the {0} in order to manage and unlock your vaults again."

Reply:
Fixed in https://github.com/cryptomator/hub/pull/258/commits/836fd8713b33e669be4a3c2621b735c5973db352 and also updated the screenshots.

Reply:
I adjusted the wording further in https://github.com/cryptomator/hub/pull/258/commits/494e14b31837da24a59449eeb8fc26bf413cdcea